builds/e2e/templates/e2e-setup.yaml

```diff
@@ -18,7 +18,8 @@ steps:
       TestPreviewIotHubConnectionString,
       TestRootCaCertificate,
       TestRootCaKey,
-      TestRootCaPassword
+      TestRootCaPassword,
+      TestBlobStoreSas
```

Comment:
This new secret needs to be documented in the "Test secrets" table in `test/README.md`.